cmd/bundle/plan.go

```diff
@@ -79,14 +80,60 @@ func newPlanCommand() *cobra.Command {
 			return root.ErrAlreadyPrinted
 		}
 
+		// Count actions by type and collect formatted actions
+		createCount := 0
+		updateCount := 0
+		deleteCount := 0
+		changed := make(map[string]bool)
+
+		for _, change := range plan.GetActions() {
+			changed[change.Group+"."+change.Key] = true
```

Comment:
This seems top be using outdated API - Group/Key are replaced by ResourceKey in https://github.com/databricks/cli/pull/3627